### PR TITLE
feat(examples): transfer transaction

### LINKED ISSUE #8: Make transaction example

### PR DESCRIPTION
**Pull request by kehiy**
- Fixes #8

### CODE REVIEW

File: examples/example_transfer_transaction.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    receiver = Address.from_string("tpc1zk4eztwv3fs6l9g776cfwkgujham79hrrwn82q5")
+
+    sender = Address.from_string("tpc1z4rlzvq8lv92cnp0k3mk5jfr00kjnz0qvklvn3u")
+    sec = PrivateKey.from_string(
+        "tsecret1pzf33h72n9phxzaty5urh5ss4m33vvdfawyvesl5jtlt9a00tnwkqyngm6z"
+    )
+
+    tx = Transaction.create_transfer_tx(lock_time, sender, receiver, amount, fee, memo)
+    tx.sign(sec)
+    # 0001b8791a0080ade2041a5468697320697320612074657374207472616e7
```
Comment (b00f) [inline examples/example_transfer_transaction.py:24]: print it

Comment (b00f) [reply examples/example_transfer_transaction.py:24]: And verify it...

Comment (kehiy) [reply examples/example_transfer_transaction.py:24]: If we print it, we will have a linter error.

File: examples/example_key_generation.py [hunk 2]
```diff
@@ -36,17 +36,24 @@ def main() -> None:
             ikm = secrets.token_bytes(32)
             sec = PrivateKey.key_gen(ikm)
             pub = sec.public_key()
-            pub.validator_address()
+            addr = pub.validator_address()
```
Comment (b00f) [inline examples/example_key_generation.py:39]: Good job, Then we can delete lines 30, 31

**Pull request merged**